packages/editor/src/core/sub-document/editor.tsx

```diff
@@ -56,7 +56,9 @@ export function SubDocumentEditor({ id, pluginProps }: SubDocumentProps) {
       // we overwrite it here (because it's probably because of tab navigation)
       setTimeout(() => {
         // fixes a bug in table plugin with disappearing buttons
-        if (e.target.nodeName?.toLowerCase() === 'button') return
+        if (['button', 'select'].includes(e.target.nodeName?.toLowerCase())) {
+          return
+        }
```

Comment:
@elbotho This is the part that fixes the focus bug. Do you see any potential problems with this change?

Reply:
Can't think of an obvious problem.
Let's see if we encounter some and add tests accordingly.